doc/firstWeek summary.md

```diff
@@ -0,0 +1,25 @@
+## First week summary with question and answers.
+
+### Question- Why OAM users needs GItOps? 
+
+> OAM users needs GitOps because of multiple reasons some of them listed below -
+- <b>Deploy Faster More Often</b> When we say “high velocity” we mean that every product team can safely ship updates many times a day — deploy instantly, observe the results in real time, and use this feedback to roll forward or back.
+- <b>Easy and Fast Error Recovery</b> — The Git record is then not just an audit log but also a transaction log. You can roll back & forth to any snapshot.
+GitOps you have a complete history of how your environment changed over time. This makes error recovery as easy as issuing a git revert and watching your environment being restored.
+
+- <b>Easier Credential Management</b> — GitOps allows you to manage deployments completely from inside your environment. For that, your environment only needs access to your repository and image registry. That’s it. You don’t have to give your developers direct access to the environment.
```

Comment:
Credential Management is not something that GitOps does. Please specify the benefit that GitOps provides instead. For example, GitOps provide isolation to different roles and environments. GitOps keeps developers from direct access to production environments.